src/main/java/org/apache/mesos/hdfs/state/LiveState.java

```diff
@@ -71,25 +73,23 @@ public void updateTaskForStatus(Protos.TaskStatus status) {
       // If initializing the first NN or reconciling the first NN or bootstrapping the first NN
       // set the status to initialized
       if (status.getMessage().equals(HDFSConstants.NAME_NODE_INIT_MESSAGE)
```

Comment:
Yes... removing them makes the most sense :)  however there is a lot of logic in the organization of the if case.   Without proper tests, I'm hesitate to remove them myself.     I'll look through it again.